pages/badge/badge.css

```diff
@@ -0,0 +1,48 @@
+@import url(../avatar/avatar.css);
+.badge {
```

Comment:
**Fix:** You can change the color of the text inside the badge to something light, as black on red is creating contrast issues.
![Screenshot 2022-01-28 at 20 42 30](https://user-images.githubusercontent.com/62737267/151571989-97eebf15-7588-45d6-ac69-d0fc4d6052de.png)

Reply:
thanks you for the feedback will impliment this change right away